### PR TITLE
update csharp admin, +tg123, -krabhishek8260

### PR DESCRIPTION
**Pull request by tg123**

### CODE REVIEW

**Comment by tg123** [comment]
cc
  @cblecker
  @fejta
  @idvoretskyi
  @mrbobbytables
  @nikhita
  @spiffxp
  @ameukam
  @palnabarun
  @savitharaghunathan

**Comment by cblecker** [comment]
This needs sign off from the SIG first.

**Comment by tg123** [comment]
ping @brendandburns

File: config/kubernetes-client/org.yaml [hunk 1]
```diff
@@ -81,6 +81,7 @@ teams:
     members:
     - brendandburns
     - krabhishek8260
```
Comment (brendandburns) [inline config/kubernetes-client/org.yaml:83]: Remove @krabishek8260 he hasn't been involved in a long time.

**Comment by brendandburns** [comment]
This looks good to me @tg123 is very active in this repo and has been a maintainer for a while.

One note to remove another member who has not been active recently.

**Comment by tg123** [comment]
@palnabarun could you please take look another time. let me know if anything missing

**Review by nikhita — APPROVED** [review]
Approving based on @brendandburns 's lgtm.

/lgtm
/approve

**Pull request merged**